### PR TITLE
Added ProgressCell to Wpf, WinForms, Gtk, Mac

### PR DESCRIPTION
**Pull request by BaseCrosser**
I also added a progress cell column to the gridview section in the test application.

@cwensley I made a PR.

### CODE REVIEW

File: Source/Eto.Wpf/Forms/Cells/ProgressCellHandler.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+		public ProgressCellHandler()
+		{
+			Control = new Column { Handler = this };
+		}
+	}
+}
+
+
+namespace Test
```
Comment (cwensley) [inline Source/Eto.Wpf/Forms/Cells/ProgressCellHandler.cs]: Oops, didn't see this before.. Could you move the class out of the Test namespace?

Comment (BaseCrosser) [inline Source/Eto.Wpf/Forms/Cells/ProgressCellHandler.cs]: I forgot I did that. I'll do it now.

**Comment by BaseCrosser** [comment]
I've done it @cwensley . Thanks for pointing that out.

**Comment by BaseCrosser** [comment]
Could you give me a hand with why it's failing? I'm still new to this..

**Comment by cwensley** [comment]
Not a problem - you just need to update the net40 csproj files to include the new files as well.  You can load use the Eto - net40.sln to test that out.

**Comment by BaseCrosser** [comment]
Thankyou, I've done it. It should build now I think.

**Comment by BaseCrosser** [comment]
I fixed the problem in Mac64. Couldn't convert double to float when making my progress bar rectangle.
Adding something to a project is a bit harder than editing something I learned now XD :P

**Comment by cwensley** [comment]
Looks great!  Thank you yet again (;

**Comment by BaseCrosser** [comment]
@cwensley You are welcome. I'm having fun with this plus I use Eto sometimes. I am just glad that I can contribute and also add what I need.

**Pull request merged**